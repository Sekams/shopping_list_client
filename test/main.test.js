import React from 'react';
import Main from '../src/components/main'

test("renders the main component", () => {
    const wrapper = shallow(
        <Main />
    );
    expect(wrapper).toMatchSnapshot();
});