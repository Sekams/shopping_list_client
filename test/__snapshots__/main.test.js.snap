// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the main component 1`] = `
<main
  role="main"
>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/login"
    />
    <Route
      component={[Function]}
      path="/register"
    />
  </Switch>
</main>
`;
